### PR TITLE
emf example: add note on FLUENT_HOST

### PR DESCRIPTION
**Pull request by PettitWesley**
Signed-off-by: Wesley Pettit <wppttt@amazon.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/fluent-bit/cloudwatchlogs-emf/README.md [hunk 1]
```diff
@@ -77,9 +77,24 @@ And then setup a port mapping on the FireLens container:
...
+```
+"environment": [
+			{
+                "name": "AWS_EMF_AGENT_ENDPOINT",
+                "value": "tcp://fluentbit:25888"
+            }
+			],
+```
+
+It is not possible to reference the value of the `FLUENT_HOST` environment variable in the value of this environment variable. However, if you are constructing EMF directly in your application code and sending it over TCP instead of using the AWS EMF libraries, using `FLUENT_HOST` may be a desirable option.
```
Comment (matthewfala) [inline examples/fluent-bit/cloudwatchlogs-emf/README.md]: Is FLUENT_HOST an IP address, or is it just tcp://fluentbit:25888

File: examples/fluent-bit/cloudwatchlogs-emf/README.md [hunk 2]
```diff
@@ -77,9 +77,24 @@ And then setup a port mapping on the FireLens container:
...
+
+The above linked steps and the bridge network mode example task definition in this tutorial use a Docker link to create a hostname in the app container for your Fluent Bit/FireLens container. It should be noted that [FireLens will set a hostname in app containers for Fluent Bit](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_firelens.html) called `FLUENT_HOST`. However, the AWS EMF libraries require an environment variable to reach the collection agent:
+
+```
+"environment": [
+			{
+                "name": "AWS_EMF_AGENT_ENDPOINT",
+                "value": "tcp://fluentbit:25888"
+            }
+			],
```
Comment (matthewfala) [inline examples/fluent-bit/cloudwatchlogs-emf/README.md]: Formatting issues.

Comment (matthewfala) [reply examples/fluent-bit/cloudwatchlogs-emf/README.md]: Where did the port come from?

I know the "fluentbit" hostname comes from the docker link.

File: examples/fluent-bit/cloudwatchlogs-emf/README.md [hunk 3]
```diff
@@ -77,9 +77,24 @@ And then setup a port mapping on the FireLens container:
               }],
 ```
 
-See the full task definition in the [emf-over-tcp](emf-over-tcp) directory.
+See the full task definition in the [emf-over-tcp](emf-over-tcp) directory. The setup steps here are again similar to using the [CW Agent to collect EMF in ECS](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Generation_CloudWatch_Agent.html).
```
Comment (matthewfala) [inline examples/fluent-bit/cloudwatchlogs-emf/README.md:84]: emf-over-tcp link is broken

**Review by matthewfala — CHANGES_REQUESTED** [review]
Minor formatting issue, broken link, and clarification question that probably would be answered with the link not being broken

File: examples/fluent-bit/cloudwatchlogs-emf/README.md (hunk 1 above)
Comment (PettitWesley) [reply examples/fluent-bit/cloudwatchlogs-emf/README.md]: its an IP

File: examples/fluent-bit/cloudwatchlogs-emf/README.md (hunk 2 above)
Comment (PettitWesley) [reply examples/fluent-bit/cloudwatchlogs-emf/README.md]: good point. Will note its the port FB is configured to listen on for TCP messages

File: examples/fluent-bit/cloudwatchlogs-emf/README.md (hunk 3 above)
Comment (PettitWesley) [reply examples/fluent-bit/cloudwatchlogs-emf/README.md:84]: it works for me when I click view file

**Comment by PettitWesley** [comment]
@matthewfala updated

File: examples/fluent-bit/cloudwatchlogs-emf/README.md (hunk 3 above)
Comment (matthewfala) [reply examples/fluent-bit/cloudwatchlogs-emf/README.md:84]: It works! Thank you.

File: examples/fluent-bit/cloudwatchlogs-emf/README.md [hunk 4]
```diff
@@ -53,33 +53,52 @@ First, we create a Fluent Bit TCP input and cloudwatch output for the EMF:
...
-              }]
+    "name": "AWS_EMF_AGENT_ENDPOINT",
+    "value": "tcp://fluentbit:25888"
+}]
 ```
 
+The value `tcp://fluentbit:25888` comes from:
+1. `fluentbit` hostname is set from the docker link.
+2. Port `25888` is the port the Fluent Bit TCP port is configured to listen on.
+
```
Comment (matthewfala) [inline examples/fluent-bit/cloudwatchlogs-emf/README.md:74]: This is very clear

**Review by matthewfala — APPROVED** [review]
Added text makes it very clear. Link works.

**Pull request merged**